app/controllers/schemes_controller.rb

```diff
@@ -291,10 +291,6 @@ def find_resource
     @scheme
   end
 
-  def user_allowed_action?
```

Comment:
I don't believe this is used anywhere, it's not a rails method or anything so I think we should remove it

Reply:
Yeah, I think we were using it to authorise in the past. Good spot